Automattic/jetpack
# Blocks: add a new Google Docs embed block

#### Changes proposed in this Pull Request:

* Adds a new block to Jetpack which allows for embedding of google documents, spreadsheets and slideshows.

#### Jetpack product discussion

* Internal reference: pdqkMK-ch-p2
 
#### Does this pull request change what data or activity we track or use?

* No.

#### Testing instructions:

* Create a new post.
* Search for available posts.
* "Google Docs", "Google Sheets" and "Google Slides" should all be available options for blocks
* Add each block variation to your post, and populate the blocks with links to publicly available documents.
  * If the document is not public, a warning should display in the editor.
* Publish the post and view the front end, the documents should be visible to the end user.
